samples/CSharp/SourceGenerators/SourceGeneratorSamples/MustacheGenerator.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+#pragma warning disable IDE0008 // Use explicit type
```

Comment:
📝 This suppression should be removed and the diagnostics fixed before this is merged